bcda/cli.go

```diff
@@ -113,14 +117,20 @@ func importCCLF9(filePath string) error {
 	for sc.Scan() {
 		b := sc.Bytes()
 		if len(bytes.TrimSpace(b)) > 0 {
-			fmt.Printf("\nXREF: %s\n", b[0:1])
-			fmt.Printf("Current identifier: %s\n", b[currIDStart:currIDEnd])
-			fmt.Printf("Previous identifier: %s\n", b[prevIDStart:prevIDEnd])
-			fmt.Printf("Previous identifier effective date: %s\n", b[prevIDEffDateStart:prevIDEffDateEnd])
-			fmt.Printf("Previous identifier obsolete date: %s\n", b[prevIDObsDateStart:prevIDObsDateEnd])
+
+			cclf9 := models.CCLF9{
```

Comment:
Do the IDs need to be associated with an ACO?  I think we can get the ACO ID from the filename.  Is there value in storing it?

Reply:
I can store it from the filename no problem. Have to think on what this means...but will add it if this is what we want.

Reply:
Trying to think about what this means.  Will we be able to do what we need to do if the ACO ID is _not_ associated with the ID references which have changed?

Reply:
^ I think this is the only open action.  Do we need ACO ID in the DB?  It can't hurt, right?  @embh @knollfear Thoughts?

Reply:
I can't think of how we'd use the ACO ID, because the beneficiaries' identifiers should be considered changed across the board, not just in the context of that ACO. Since we would be able to get it later from the filename, not sure it's worth having separated out. But it can't hurt. We will be saving ACO ID in its own column for CCLF8 files, if that makes any difference.

Reply:
yeah to what @embh said, the aco id would go "wherever we decided for the overall ingest" but we could separate it out if that's what we want to do.

Reply:
To follow up on this: to be able to use the filename to get the ACO ID, we need the CCLF9 xref records to be associated with a file, so we need the ID from the file in this model.